src/protocols/coda_pow.ml

```diff
@@ -967,6 +967,120 @@ module type External_transition_intf = sig
   val staged_ledger_diff : t -> staged_ledger_diff
 end
 
+module type External_transition_validation_intf = sig
+  type state_hash
+
+  type external_transition
+
+  type staged_ledger
+
+  type staged_ledger_error
+
+  type transition_frontier
+
+  module Truth : sig
```

Comment:
can you lift this into a separate library?

Reply:
and we can just have a direct dependency on that in coda_pow (and everywhere)

Reply:
done